### PR TITLE
[CMake] Enable Protobuf based readers for Android

### PR DESCRIPTION
**Pull request by dkurt**
Android toolchain is able build ONNX/PDPD/TF readers.

test build: https://github.com/dkurt/openvino_java/actions/runs/3043828663/jobs/4903532041

### CODE REVIEW

File: cmake/features.cmake [hunk 1]
```diff
@@ -144,7 +144,7 @@ ie_dependent_option (ENABLE_GPU_DEBUG_CAPS "enable GPU debug capabilities at run
 
 ie_dependent_option (ENABLE_CPU_DEBUG_CAPS "enable CPU debug capabilities at runtime" ON "ENABLE_DEBUG_CAPS" OFF)
 
-if(ANDROID OR WINDOWS_STORE OR (MSVC AND (ARM OR AARCH64)))
```
Comment (ilya-lavrenov) [inline cmake/features.cmake:147]: do we need it for 2022.2? It's already CF
for master we have fixed this and protobuf readers are enabled unconditionally

Comment (dkurt) [reply cmake/features.cmake:147]: I see. Let's close PR.

Comment (ilya-lavrenov) [reply cmake/features.cmake:147]: BTW, you can see it here https://github.com/openvinotoolkit/openvino/pull/12881

Comment (dkurt) [reply cmake/features.cmake:147]: Great! Thank you

**Pull request closed without merging**